npm-shrinkwrap.json

```diff
@@ -8564,6 +8564,12 @@
     },
     "blob": {
       "version": "0.0.4"
+    },
+    "ms": {
```

Comment:
Should this have been committed in a previous pull request? Not seeing how this is related to the dependency added here.